examples/custom_secrets/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "region" {
+  description = "The region to deploy AWS resources to."
+  type        = string
+}
+
+variable "secret_path_format" {
+  description = "The custom secret path to use."
+  type        = string
+}
+
+variable "secrets_store_enabled" {
+  description = "Whether or not to write to the secrets store."
+  type        = bool
+  default     = true
+}
+
+variable "secrets_store_type" {
+  description = "The secrets store type to use."
```

Comment:
```suggestion
  description = "The secrets store type to use., `ssm` or `asm`"
```

Reply:
maybe add `default = "SSM` so we don't have to add it to the examples where `secrets_store_enabled = false`